docs/src/tutorials/mass_spring_damper.md

```diff
@@ -0,0 +1,125 @@
+# Translational Mass Spring Damper System
+
+This tutorial demonstrates the response of a mass-spring-damper system. Rather than applying an external force, the mass is initialized with some positional offset from equilibrium and some non-zero velocity (system responses under such conditions are commonly referred to as "unforced" responses). Assuming a lossless system, a mass-spring system in the absence of a damper will continually oscillate at some natural frequency. The addition of a damper will result in (1) underdamped, (2) overdamped, or (3) critically damped motion, depending on the relative values of our mass (`m`), spring constant (`c`), and damping constant (`d`). In this first example, we will demonstrate an underdamped system. Values for `m`, `c`, and `d` were chosen such that the roots of the polynomial `s^2 + d/m * s + c/m` are complex conjugates (the equivalent condition for underdamped motion is `d < sqrt(4 * m * c)`). In the case of underdamped systems, our mass will undergo damped oscillations. This damping corresponds to system energy loss, and the amplitudes of said oscillations will decrease towards zero as time approaches infinity.
+
+Let's start by importing what we need.
+
+```@example
+using ModelingToolkitStandardLibrary.Mechanical.Translational, ModelingToolkit, OrdinaryDiffEq, ControlSystems, Plots
+import ModelingToolkitStandardLibrary.Blocks
+```
+
+Now let's initialize our system state and define our components and connections. Here, `s0` is the fixed offset position of our flange housing, `s_rel0` is our unstretched spring length, and `s_start` and `v_start` are the initial values of the absolute position and linear velocity of our sliding mass, respectively. Finally, `m`, `c`, and `d` are our mass, spring, and damping constants, respectively. All units are in SI units.
+
+```@parameters t
+D = Differential(t)
+
+s0 = 4.5
+s_rel0 = 1
+s_start = 3
+v_start = 10
+m = 1
+c = 10
+d = 1
+
+@named fixed = Fixed(s0=s0)
+@named mass = Mass(m=m, s_start=s_start, v_start=v_start)
+@named damper = Damper(d=d)
+@named spring = Spring(c=c, s_rel0=s_rel0)
```

Comment:
```suggestion
@named fixed = Fixed(; s0)
@named mass = Mass(; m, s_start, v_start)
@named damper = Damper(; d)
@named spring = Spring(; c, s_rel0)
```
In case you didn't know, if your variable shares the same name as a keyword argument (a common case), you do not need to repeat the name. You do need to have the semi-colon first though to indicate that keyword arguments are coming.